src/status_im/ui/screens/discover/subs.cljs

```diff
@@ -2,18 +2,25 @@
   (:require [re-frame.core :refer [reg-sub]]
             [status-im.utils.datetime :as time]))
 
-(defn- calculate-priority [{:keys [chats current-public-key]
-                            :contacts/keys [contacts]}
+(reg-sub :get-discoveries :discoveries)
```

Comment:
can you remove this file from your commit so that there is no conflict with #2483 ?

Reply:
Sure